Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -297,6 +297,15 @@ public async Task SetDisplaying( bool displaying )
         await ParentDataGrid.Refresh();
     }
 
+    public async Task SetDisplayOrder( int displayOrder, bool forceParentRefresh = false )
+    {
+        DisplayOrder = displayOrder;
+        await ParentDataGrid.ColumnDisplayOrderChanged.InvokeAsync( new ColumnDisplayOrderChangedEventArgs<TItem>( this, displayOrder ) );

```

Comment:
Wouldn't be better to make the EventCallback for the whole reorder batch?
Because with this users will need to keep track of the columns order anyway. What about just pushing all the columns with their order?? Or only the change ones?
idk. just an idea